sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -54,6 +54,42 @@ def agg(self, *args, **kwargs):
       'order-sensitive')
   diff = frame_base.wont_implement_method('order-sensitive')
 
+  @frame_base.args_to_kwargs(pd.Series)
+  def nlargest(self, **kwargs):
+    if 'keep' in kwargs and kwargs['keep'] != 'all':
+      raise frame_base.WontImplementError('order-sensitive')
+    per_partition = expressions.ComputedExpression(
+        'nlargest',
```

Comment:
Would using 'nlargest' for the name of both the per-partition and global computed expression cause confusion for debugging or UX?

Reply:
Good point, updated.